tests/python/contrib/test_cmsisnn/test_pooling.py

```diff
@@ -45,11 +45,15 @@ def make_model(
     zero_point=-33,
     relu_type="RELU",
     layout="NHWC",
+    input_op=None,
```

Comment:
I assume this is because we're creating an object and it doesn't like to use that as a default?

Reply:
I was more wondering why we can't do `input_op=relay.var(...)` but I think that's linted against because Python would only instantiate the argument once?

Reply:
oh, I didn't think of it. Let me try.

Reply:
It errors out on shape and dtype which are passed as another set of the same func's arguments.

Reply:
👍